pkg/handlers/generic/mutation/containerdunprivilegedports/unprivileged_ports.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2024 Nutanix. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+package containerdunprivilegedports
+
+import (
+	_ "embed"
+	"path"
+
+	cabpkv1 "sigs.k8s.io/cluster-api/bootstrap/kubeadm/api/v1beta1"
+)
+
+const (
+	// TODO Factor out this constant to a common package.
```

Comment:
Is this PR enough https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/pull/656/files?

Reply:
Good catch! Thanks! Updated to use new common function to derive patch file path.